server/src/main/java/com/cloud/consoleproxy/ConsoleProxyManagerImpl.java

```diff
@@ -238,17 +238,20 @@
     private boolean _sslEnabled = false;
     private String _consoleProxyUrlDomain;
 
-    // global load picture at zone basis
     private SystemVmLoadScanner<Long> _loadScanner;
-    private Map<Long, ZoneHostInfo> _zoneHostInfoMap; // map <zone id, info about running host in zone>
-    private Map<Long, ConsoleProxyLoadInfo> _zoneProxyCountMap; // map <zone id, info about proxy VMs count in zone>
-    private Map<Long, ConsoleProxyLoadInfo> _zoneVmCountMap; // map <zone id, info about running VMs count in zone>
+    private Map<Long, ZoneHostInfo> _zoneHostInfoMap;
+    private Map<Long, ConsoleProxyLoadInfo> _zoneProxyCountMap;
+    private Map<Long, ConsoleProxyLoadInfo> _zoneVmCountMap;
```

Comment:
while touching this anyway, we might want to remove the silly undescores.

Reply:
@DaanHoogland done. I removed all initial underscore of variables and renamed some. Also, as it was used in `ConsoleProxyManagerTest`, I changed it too.

Reply:
you are taking a radical aproach to my comment. I applaud you for that :clap: .  I'll run the smoke tests again just to be sure.